examples/ez-food/order-processor/order_details_listener.bal

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import orderprocessor.store;
+
+import ballerinax/java.jms;
+
+const string ORDERS_QUEUE = "orders";
+const string ORDER_CONFIRMATIONS_QUEUE = "order-confirmation";
+
+configurable jms:ConnectionConfiguration activeMqConnectionConfig = {
+    initialContextFactory: "org.apache.activemq.jndi.ActiveMQInitialContextFactory",
+    providerUrl: "tcp://localhost:61616"
+};
+
+final jms:MessageProducer producer = check createProducer();
+
+isolated function createProducer() returns jms:MessageProducer|error {
+    jms:Connection connection = check new (activeMqConnectionConfig);
+    jms:Session session = check connection->createSession();
+    return session.createProducer();
+}
+
+service "order-details-receiver" on new jms:Listener(
+    connectionConfig = activeMqConnectionConfig,
+    consumerOptions = {
+        destination: {
+            'type: jms:QUEUE,
+            name: ORDERS_QUEUE
+        }
+    }
+) {
```

Comment:
Shall we exact out the listener declaration to make it more readable?

Reply:
Fixed with [fe44bb282317acb121f3943bd944ac24c8059ec1](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/119/commits/fe44bb282317acb121f3943bd944ac24c8059ec1)